### PR TITLE
chore(j-s): Add fields to case_file table

### PR DESCRIPTION
**Pull request by oddsson**
# Add fields to case_file table

[Asana](https://app.asana.com/0/1199153462262248/1203127878754336/f)

## What

Add fields `policeCaseNumber`, `userGeneratedFilename`, `chapter`, and `orderWithinChapter` to the case_file table in DB.

## Why

This is needed for the caseFile feature we are building.

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/judicial-system/backend/migrations/20221011135323-update-case.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+          },
+          {
+            transaction,
+          },
+        ),
+      ]),
+    )
+  },
+
+  down: (queryInterface, Sequelize) => {
```
Comment (gudjong) [inline apps/judicial-system/backend/migrations/20221011135323-update-case.js:55]: Consider removing `Sequelize` as it is not used

**Review by gudjong — CHANGES_REQUESTED** [review]

File: libs/judicial-system/types/src/lib/file.ts [hunk 2]
```diff
@@ -63,6 +63,10 @@ export interface CaseFile {
   key?: string
   size: number
   category?: CaseFileCategory
+  policeCaseNumber?: string
+  userGeneratedFilename?: string
+  chapter?: number
+  orderWithinChapter?: number
 }
 
 export interface CreateFile {
```
Comment (Arnarkari93) [inline libs/judicial-system/types/src/lib/file.ts:72]: `policeCaseNumber` missing here

**Review by Arnarkari93 — COMMENTED** [review]
Looks good, I think here is only one place left.

**Review by Arnarkari93 — APPROVED** [review]

File: libs/judicial-system/types/src/lib/file.ts [hunk 3]
```diff
@@ -63,10 +63,17 @@ export interface CaseFile {
...
+  caseId: string
```
Comment (Arnarkari93) [inline libs/judicial-system/types/src/lib/file.ts:73]: caseId is not part of the CreateFile.dto. It can be removed here. 
```suggestion
```

**Review by gudjong — APPROVED** [review]

**Pull request merged**